### DISCUSSION TITLE
copy file from SDcard on ESP32 to PC

### DISCUSSION THREAD

**Discussion by robhamerling** [micropython/micropython#9615] [Using MicroPython] [answered]
With mpremote it is easy to copy files from ESP32 to PC, but how to copy files which are located on an SDCard on the  ESP32?

**Comment by jimmo** [comment, marked as answer]
@robhamerling The same way -- mpremote accesses the device filesystem via VFS, so use the same path that the device uses. e.g. if the sdcard is mounted at /sdcard then `mpremote cp :/sdcard/file.txt file.txt` should work.

**Reply by robhamerling** [reply]
As far as I understand it "mounted" is the clue. The card is physically present and I can mount it with a Python script, but I do not think it remains mounted after the script terminated.

**Reply by jimmo** [reply]
As long as you mount it in boot.py, this should work.

**Reply by robhamerling** [reply]
Great, it works! Nice assist,  thanks for the suggestion.
